helm-chart/ray-cluster/values.yaml

```diff
@@ -100,6 +102,60 @@ worker:
     - mountPath: /tmp/ray
       name: log-volume
 
+extraWorkers:
+  generic:
```

Comment:
The Helm docs do suggestion presenting lists as maps, as is done here.

However, the map key becomes redundant with the groupName. I wonder if we could drop `groupName` and add a comment that the key will be used as the groupName?

Also, could we use a more descriptive name for this workerGroup, such as small-group? 
(small, because the resource requests are quite small...which is fine for a default/example config)

Reply:
Makes sense! I used initially the groupName from the key but I switched it back to groupName var to keep it consistent with the default worker. I will add a comment and use the key instead!

Will update the worker group name also, thanks